locale/en/get-involved/events.md

```diff
@@ -7140,6 +7140,15 @@ regions:
         lon: -90.5477204
   - region: North America
     conferences:
+      - name: Node Knockout
+        dates:
+          - 2015-11-12T00:00:00.000Z
```

Comment:
I'm not even sure where these dates are used, but would you want to list the 2016 dates instead?